Makefile

```
@@ -1,30 +1,16 @@
-.PHONY: clean coldstart mrproper up build
+.PHONY: clean coldstart mrproper up build logs
 all: build up
 
-compose := docker-compose \
-	-f docker-compose.yml \
-	-f lisk-service/docker-compose.core.yml \
-	-f lisk-service/docker-compose.gateway.yml \
-	-f lisk-service/docker-compose.market.yml \
-	-f lisk-service/docker-compose.newsfeed.yml \
-	-f lisk-service/docker-compose.gateway-ports.yml
+compose := docker-compose
 
-up: up-core3
-
-up-mainnet: up-custom-core2-mainnet
-
-up-testnet: up-custom-core2-testnet
-
-up-core3: up-custom-core3-default
-
-up-custom-%:
-	cd ./docker && $(compose) --env-file ./network/$*.env up --detach
+up:
+	docker-compose up -d
```

For consistency purposes:
```suggestion
	$(compose) up -d
```